src/systems/particle.rs

```diff
@@ -0,0 +1,36 @@
+use crate::{Particle, ParticleEmitter, Position};
+use bevy::prelude::*;
+use std::time::Duration;
+
+/// emit new particles from entites with the `ParticleEmitter` component
+pub fn particle_emission_system(
+    mut commands: Commands,
+    query: Query<(&ParticleEmitter, &Sprite, &Position)>,
```

Comment:
Note to self: this should be `Query<(&Sprite, &Position), With<ParticleEmitter>>,` because I don't actually need access to the `ParticleEmitter` state